lib/pubsub/subscription.js

```diff
@@ -269,13 +272,16 @@ function Subscription(pubsub, options) {
 
   this.autoAck = is.boolean(options.autoAck) ? options.autoAck : false;
   this.closed = true;
-  this.interval = is.number(options.interval) ? options.interval : 10;
+  this.encoding = options.encoding || 'utf-8';
   this.inProgressAckIds = {};
+  this.interval = is.number(options.interval) ? options.interval : 10;
   this.maxInProgress =
     is.number(options.maxInProgress) ? options.maxInProgress : Infinity;
   this.messageListeners = 0;
   this.paused = false;
-  this.encoding = options.encoding || 'utf-8';
+  // The default timeout used in this library is 60s, but a pull request times
+  // out around 90 seconds.
+  this.timeout = is.number(options.timeout) ? options.timeout : 90000;
```

Comment:
Good idea for some padding on the timeout. I just pushed an update, but went on the side of allowing more time (92s) rather than less (88s). I figured its best to give the API more time to fulfill the request before terminating. Here's the code: https://github.com/GoogleCloudPlatform/gcloud-node/commit/a78b77d6e377e0737f549d60259669a86cbab135#diff-9c43bce0f7f5594b8154b1b8a747af5aR286

LMK if you think 88 is still the right way to go.